### DISCUSSION TITLE
The "original" argument must be of type Function"

### DISCUSSION THREAD

**Discussion by christianangon** [bostrom/text-to-image#222] [Q&A]
Hi! Can some help me. Whenever i tried to test the it out, it always says that "The "original" argument must be of type Function""

![image](https://user-images.githubusercontent.com/92027960/165043980-a65c2969-0a25-461d-88ec-1f64cecaaab5.png)
![image](https://user-images.githubusercontent.com/92027960/165044011-e27b754e-6388-4883-997a-761ca1564121.png)

**Comment by bostrom** [comment]
Hm, it doesn't sound like it's coming from `text-to-image`, there's no `original` argument in there. Can you try to make a minimal vue application without all the other dependencies and see if it occurs then too? Or make a small [CodeSandbox vue app](https://codesandbox.io/s/vue)?
